tools/plain_train_net.py

```diff
@@ -153,8 +153,9 @@ def do_train(cfg, model, resume=False):
             storage.step()
 
             loss_dict = model(data)
-            losses = sum(loss for loss in loss_dict.values())
-            assert torch.isfinite(losses).all(), loss_dict
```

Comment:
it means "print loss_dict if the assertion failed". It's not an assertion on two statements.

Reply:
You are right, I reverted the commit.